cypress/fixtures/erc20.contract

```diff
@@ -0,0 +1 @@
er's account multiple times, up to"," the `value` amount.",""," If this function is called again it overwrites the current allowance with `value`.",""," An `Approval` event is emitted."],"label":"approve","mutates":true,"payable":false,"returnType":{"displayName":["Result"],"type":8},"selector":"0x681266a0"},{"args":[{"label":"from","type":{"displayName":["AccountId"],"type":2}},{"label":"to","type":{"displayName":["AccountId"],"type":2}},{"label":"value","type":{"displayName":["Balance"],"type":0}}],"docs":[" Transfers `value` tokens on the behalf of `from` to the account `to`.",""," This can be used to allow a contract to transfer tokens on ones behalf and/or"," to charge fees in sub-currencies, for example.",""," On success a `Transfer` event is emitted.",""," # Errors",""," Returns `InsufficientAllowance` error if there are not enough tokens allowed"," for the caller to withdraw from `from`.",""," Returns `InsufficientBalance` error if there are not enough tokens on"," the account balance of `from`."],"label":"transfer_from","mutates":true,"payable":false,"returnType":{"displayName":["Result"],"type":8},"selector":"0x0b396f18"}]},"storage":{"struct":{"fields":[{"layout":{"cell":{"key":"0x0000000000000000000000000000000000000000000000000000000000000000","ty":0}},"name":"total_supply"},{"layout":{"cell":{"key":"0x0100000000000000000000000000000000000000000000000000000000000000","ty":1}},"name":"balances"},{"layout":{"cell":{"key":"0x0200000000000000000000000000000000000000000000000000000000000000","ty":6}},"name":"allowances"}]}},"types":[{"id":0,"type":{"def":{"primitive":"u128"}}},{"id":1,"type":{"def":{"composite":{"fields":[{"name":"offset_key","type":5,"typeName":"Key"}]}},"params":[{"name":"K","type":2},{"name":"V","type":0}],"path":["ink_storage","lazy","mapping","Mapping"]}},{"id":2,"type":{"def":{"composite":{"fields":[{"type":3,"typeName":"[u8; 32]"}]}},"path":["ink_env","types","AccountId"]}},{"id":3,"type":{"def":{"array":{"len":32,"type":4}}}},{"id":4,"type":{"def":{"primitive":"u8"}}},{"id":5,"type":{"def":{"composite":{"fields":[{"type":3,"typeName":"[u8; 32]"}]}},"path":["ink_primitives","Key"]}},{"id":6,"type":{"def":{"composite":{"fields":[{"name":"offset_key","type":5,"typeName":"Key"}]}},"params":[{"name":"K","type":7},{"name":"V","type":0}],"path":["ink_storage","lazy","mapping","Mapping"]}},{"id":7,"type":{"def":{"tuple":[2,2]}}},{"id":8,"type":{"def":{"variant":{"variants":[{"fields":[{"type":9}],"index":0,"name":"Ok"},{"fields":[{"type":10}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":9},{"name":"E","type":10}],"path":["Result"]}},{"id":9,"type":{"def":{"tuple":[]}}},{"id":10,"type":{"def":{"variant":{"variants":[{"index":0,"name":"InsufficientBalance"},{"index":1,"name":"InsufficientAllowance"}]}},"path":["erc20","erc20","Error"]}},{"id":11,"type":{"def":{"variant":{"variants":[{"index":0,"name":"None"},{"fields":[{"type":2}],"index":1,"name":"Some"}]}},"params":[{"name":"T","type":2}],"path":["Option"]}}]}}
```

Comment:
We should think about how to integrate everything in the Ci in a way that the CI fetches the latest version of ink! examples from the ink! Repo, builds them and then runs Cypress tests against them. But this is just a future suggestion, for now its good to go, I guess.